tools/flowc/incremental/fi_helpers.flow

```diff
@@ -558,3 +569,52 @@ fiTypeIsFlow(t : FiType) {
 		default: false;
 	}
 }
+
+fiModuleDecls(m : FiModule) -> [FiDeclaration] {
+	decls = ref [];
+	iter(m.functions, \f -> refArrayPush(decls, f));
```

Comment:
This has terrible performance and is imperative. Use concatA instead:
  concatA([m.functions, m.globalVars, m.natives])
Shorter, quicker, functional. Same below.

Reply:
Fixed.